test/mailers/previews/report_mailer_preview.rb

```diff
@@ -0,0 +1,5 @@
+class ReportMailerPreview < ActionMailer::Preview
+  def first_monthly_report
+    ReportMailer.first_report_of_month(Report.last, User.where(admin: true).first)
```

Comment:
Changed to:
`ReportMailer.first_report_of_month(Report.last, User.admins.first)`